### PR TITLE
perf(local debug): update local debug to separate aad

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: packages/fx-core/src/common/local/projectSettingsHelper.ts [hunk 1]
```diff
@@ -41,8 +42,9 @@ export class ProjectSettingsHelper {
 
   public static includeAAD = (projectSettings: ProjectSettings | undefined): boolean =>
     !ProjectSettingsHelper.isMigrateFromV1(projectSettings) &&
-    (projectSettings?.solutionSettings as AzureSolutionSettings)?.hostType ===
-      HostTypeOptionAzure.id;
+    (projectSettings?.solutionSettings as AzureSolutionSettings)?.activeResourcePlugins?.includes(
```
Comment (swatDong) [inline packages/fx-core/src/common/local/projectSettingsHelper.ts:45]: `ProjectSettingsHelper` follows the same logic as plugin activate.
I note that AAD plugin has comment "[// For scaffold, activeResourecPlugins is undefined](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/plugins/resource/aad/index.ts#L40)", so does here need the same logic? `ProjectSettingsHelper` is used by scaffold as well.

Comment (KennethBWSong) [reply packages/fx-core/src/common/local/projectSettingsHelper.ts:45]: The comment in AAD plugin means that when activate() is called in scaffold, activeResourcePlugins is undefined. In this case, when ProjectSettingsHelper is called, activeResourcePlugins already have values.

Comment (swatDong) [reply packages/fx-core/src/common/local/projectSettingsHelper.ts:45]: I see, make sense to me.

**Review by swatDong — APPROVED** [review]
Code logic looks good to me. UT fix needed since field changes.

**Review by swatDong — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Comment by blackchoey** [comment]
add test case to test situation that AAD plugin is not enabled?

**Comment by KennethBWSong** [comment]
> add test case to test situation that AAD plugin is not enabled?

Working on new test cases. Will add in another pr.

**Review by blackchoey — APPROVED** [review]

**Pull request merged**